lib/plugins.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+var _ = require('lodash');
+var prefix = 'gemini-';
+
+module.exports = {
+    load: function(gemini, options) {
+        var enabledPlugins = _.reject(_.pairs(options.plugins), function(p) {
```

Comment:
The step is kinda related to plan I have on making another PR to add command line parameters that you can use to enable/disable plugins. It enables you to run for certain plugins for example in CI environment, but not locally - without modifying the configuration file.

As a simple example you could have something like this in configuration:

```
plugins:
  express:
    disabled: true
    port: 1234
```

And when running `gemini test test.js` locally, the plugin is not run, but in CI environment `gemini test test.js --plugin=express` could be used.

If you want, I can remove the step for now, and include it in the upcoming PR ?